main.py

```diff
@@ -80,7 +80,11 @@
 
 from typing import Any, Dict, Optional, Text, Type # isort:skip # pylint: disable=unused-import
 
-transaction_services = models.Registry.import_transaction_services() # type: ignore[no-untyped-call]
+MYPY = False
+if MYPY:
+    from mypy_imports import * # pragma: no cover # pylint: disable=import-only-modules,wildcard-import,unused-wildcard-import
```

Comment:
Ditto as above! Also, why are we importing using `*`?

Reply:
These will only be imported during mypy checks not during runtime. So we have no issue using wildcard import.
What do you think?

Reply:
Three reviewers have already commented on not using `*`. Please don't use `*` anywhere.